package.json

```diff
@@ -81,12 +81,12 @@
     "extract-css-chunks-webpack-plugin": "^4.9.0",
     "html-webpack-plugin": "^4.0.0",
     "husky": "^3.0.0",
-    "jest": "^25.0.0",
+    "jest": "^26.0.0",
     "jest-canvas-mock": "^2.1.0",
     "jest-junit": "^13.0.0",
     "lerna": "^4.0.0",
     "lint-staged": "^9.0.2",
-    "optimize-css-assets-webpack-plugin": "^5.0.5",
```

Comment:
we can merge this but i think it won't fix the snyk issue, we need `nth-check 2.0.1`